File: src/com/falconraptor/timegame/players/Player.java
package com.falconraptor.timegame.players;

import com.falconraptor.timegame.buildings.*;
import com.falconraptor.timegame.research.*;
import com.falconraptor.timegame.resources.*;

public class Player {
	public String username, password;
	public Buildings buildings = new Buildings();
	public Resources resources = new Resources();
	public Research research = new Research();
	public Level level = new Level("1");
}

File: src/com/falconraptor/timegame/players/Player.scala
package com.falconraptor.timegame.players

import com.falconraptor.timegame.buildings._
import com.falconraptor.timegame.research._
import com.falconraptor.timegame.resources._

class Player {
  var username: String = null
  var password: String = null
  var buildings: Buildings = new Buildings
  var resources: Resources = new Resources
  var research: Research = new Research
  var level: Level = new Level("1")
}